### DISCUSSION TITLE
1850 temperatures in 112-117

### DISCUSSION THREAD

**Discussion by islasimpson** [NCAR/cesm_dev#22] [3. Calibration/tuning]
Here's a comparison of runs 112, 114, 115, 116, 117 with the Berkeley Earth Surface Temperature (BEST) observations.  I'm comparing with 1850-1879 of BEST and note that some of the other observational datasets have NaN's over a large part of the planet during this time i.e., there are likely big uncertainties in the observations.  

Here are the climatologies:

![Screenshot from 2024-11-22 17-57-09](https://github.com/user-attachments/assets/4e0bf64c-07be-4207-bddf-d00e0a067936)

Here are the anomalies from BEST:

![Screenshot from 2024-11-22 18-49-47](https://github.com/user-attachments/assets/d62ddd15-fb33-4167-b09e-5610800e5086)

For reference, here's what we had in CESM2

![Screenshot from 2024-11-22 18-50-20](https://github.com/user-attachments/assets/7872d6e3-3358-473d-9625-218464405738)

So we're too warm over the NH continents.  116 and 117 are too warm over much of the oceans, whereas the others have more mixed patterns.  None of what we have right now is any worse than what we had in CESM2.

**Comment by dlawrenncar** [comment]
@islasimpson Could you plot this for some time period around present day, at least for CESM2 and, not quite as important, any of the CESM3 beta simulations that we have for the historical period.  Or, if you know the answer already, I am curious if the land biases are roughly the same in the pre-industrial (when obs are poor) as in the present day or if they are totally different.

**Reply by islasimpson** [reply]
Here's a plot of TS minus BEST for 1979-2013.  I think the biases are pretty similar.  

<img width="868" alt="Screenshot 2024-12-08 at 7 38 14 PM" src="https://github.com/user-attachments/assets/4fe5981d-0dde-4ed6-b01a-a6ad4c04a2d4">

Over land, the more appropriate comparison with BEST is probably TREFHT.  I think BEST is surface temperature over the ocean, 2m temperature over land.  So here's a plot of TREFHT minus BEST.  Similar warm bias over land but a little muted compared to TS.  I'm surprised how different the TREFHT and TS biases are over the ocean though.  Anyway, I think the bottom line is that the same biases are there in present day.

<img width="871" alt="Screenshot 2024-12-08 at 7 40 20 PM" src="https://github.com/user-attachments/assets/199532c3-6ff3-43f8-8e55-467579d95935">

**Comment by Vlada3333** [comment]
Thank you for the information!
